Makefile.am

```diff
@@ -314,6 +318,20 @@ libscrollplugin_la_CPPFLAGS = \
 libscrollplugin_la_LIBADD = $(SCROLL_PLUGIN_LIBS)
 libscrollplugin_la_LDFLAGS = -module -avoid-version -no-undefined
 
+# # # EXAMPLES # # #
+
+noinst_PROGRAMS = eos-player
+
+eos_player_SOURCES = examples/eos-player.c
+eos_player_CPPFLAGS = \
+        $(EOS_KNOWLEDGE_PRIVATE_CFLAGS) \
```

Comment:
Mixed tabs and spaces in these two assignments — careful, they can ruin your makefiles!